checkov/terraform/checks/resource/__init__.py

```diff
@@ -1,3 +1,4 @@
 from checkov.terraform.checks.resource.aws import *
 from checkov.terraform.checks.resource.gcp import *
 from checkov.terraform.checks.resource.azure import *
+from checkov.terraform.checks.resource.github import *
```

Comment:
sweet!

Reply:
i got a little carried away here, looks like this PR also has the new provider and test for github

Reply:
np. looks awesome. would you like to merge those together? 
If you feel like the github is ready, we can merge those.

Reply:
this was supposed to be a separate PR but its all the code to add support for the github provider and its first check. Hope thats ok or i can split to another PR. Got a bit keen.